deps.json

```diff
@@ -1,20 +1,20 @@
 {
-    "github.com/ActiveState/golor": {
-        "repo": "http://git-mirrors.stackato.com/github.com/ActiveState/golor.git",
-        "version": "642e80457a",
+    "github.com/hpcloud/golor": {
+        "repo": "git@github.com:hpcloud/golor.git",
```

Comment:
Please use `https://github.com/hpcloud/golor.git` for the repo (so that it does not require SSH auth to GitHub to clone).  That works because it's a public repo and doesn't require HTTP basic auth either.